srtcore/queue.cpp

```diff
@@ -388,14 +364,47 @@ uint64_t CSndUList::getNextProcTime()
    return m_pHeap[0]->m_llTimeStamp_tk;
 }
 
+
+void CSndUList::realloc_()
+{
+   CSNode** temp = NULL;
+
+   try
+   {
+       temp = new CSNode * [m_iArrayLength * 2];
```

Comment:
Maybe remove spaces around '*'? The syntax is confusing enough, spaces make it look even more like multiplication. This is, obviously, an opinion, feel free to ignore if your opinion differs.

Reply:
A kinda standard in C++ is that there is no space between type name and `*` for pointer types (or even more precisely, to glue the asterisk rather to preceding type name than to a following symbol name - which isn't exactly the case here though). Spaces around `*` as a multiplication operator is another matter.

Reply:
Applied CLang format from #772 to this line of code.
Changed
`temp = new CSNode * [m_iArrayLength * 2];`
to
`temp = new CSNode *[2 * m_iArrayLength];`